war/src/main/webapp/css/style.css

```diff
@@ -1891,3 +1891,28 @@ body.no-sticker #bottom-sticker {
+
+.remodal {
+  padding: 0;
+  text-align: left;
+}
+.remodal-wrapper {
+  padding: 10px 10px 0;
+}
+.remodal .container {
+  padding-right: 0;
+  padding-left: 0;
+  max-width: 800px;
+}
+.row .col-md-offset-2 {
+  margin-left: 0;
+}
+.row .col-md-20 {
+  padding-left: 0;
+  padding-right: 0;
+  width: 100%
+}
```

Comment:
👎 Please, please!!! Do not override styles by the CSS framework. This is BIG hack!